internal/cli/elemental-toolkit/action/install.go

```diff
@@ -82,7 +82,7 @@ func Install(ctx *cli.Context) error { //nolint:dupl
 	return nil
 }
 
-func digestInstallSetup(s *sys.System, flags *cmd.InstallFlags) (*deployment.Deployment, error) {
+func DigestInstallSetup(s *sys.System, flags *cmd.InstallFlags) (*deployment.Deployment, error) {
```

Comment:
I think this should not be needed, relaying on `pkg/install` and `pkg/deployment` should be enough. This also implies relying on the public API.